modules/nodes-cluster-enabling-features-cluster.adoc

```diff
@@ -45,4 +46,3 @@ spec:
 <1> Add the `featureSet: "TechPreviewNoUpgrade"` parameter.
```

Comment:
line 33 : On the *Feature Gates* page, click *Create Feature Gates*.
should modified to : 
4. On the Feature Gates page, click the Feature Gates named cluster
5. On the Feature Gates cluster page, click Actions → Edit FeatureGate.
And original 5 and 6  should change to 6 and 7
(My comment can refer to : http://file.rdu.redhat.com/~ahardin/July102020/post-install-config/post_installation_configuration/cluster-tasks.html#nodes-cluster-enabling-features-cluster_post-install-cluster-tasks

Reply:
I was able to follow the procedure as stated.